extensions/azurePublish-new/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "azure-publish-new",
+  "version": "1.0.0",
+  "description": "Publish bot to an Azure Web App",
+  "main": "dist/extension.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "node ../scripts/build.js",
+    "watch": "yarn build --watch"
+  },
+  "composer": {
+    "enabled": true,
+    "bundles": [
+      {
+        "id": "azurePublishNew",
+        "path": "dist/publish.js"
+      }
+    ],
+    "contributes": {
+      "views": {
+        "publish": [
+          {
+            "bundleId": "publish"
+          }
+        ]
+      }
+    }
+  },
+  "dependencies": {
+    "@azure/arm-appinsights": "^3.0.0",
+    "@azure/arm-appservice": "^6.0.0",
+    "@azure/arm-appservice-profile-2019-03-01-hybrid": "1.0.0",
+    "@azure/arm-botservice": "2.0.0",
+    "@azure/arm-cognitiveservices": "^5.2.0",
+    "@azure/arm-cosmosdb": "^9.0.0",
+    "@azure/arm-deploymentmanager": "3.0.0",
+    "@azure/arm-keyvault": "^1.2.1",
+    "@azure/arm-keyvault-profile-2020-09-01-hybrid": "^1.0.0",
+    "@azure/arm-resources": "2.1.0",
+    "@azure/arm-search": "^1.3.0",
+    "@azure/arm-storage": "^15.0.0",
+    "@azure/arm-subscriptions": "2.0.0",
+    "@azure/cognitiveservices-luis-authoring": "4.0.0-preview.1",
+    "@azure/cosmos": "^3.7.4",
+    "@azure/graph": "5.0.1",
+    "@azure/keyvault-secrets": "^4.1.0",
+    "@azure/ms-rest-browserauth": "0.1.4",
+    "@azure/ms-rest-js": "^2.0.7",
+    "@azure/ms-rest-nodeauth": "3.0.3",
+    "@bfc/built-in-functions": "../../Composer/packages/tools/built-in-functions",
+    "@bfc/code-editor": "../../Composer/packages/lib/code-editor",
+    "@bfc/extension-client": "file:../../Composer/packages/extension-client",
+    "@bfc/indexers": "../../Composer/packages/lib/indexers",
+    "@bfc/shared": "../../Composer/packages/lib/shared",
+    "@bfc/ui-shared": "../../Composer/packages/lib/ui-shared",
+    "@emotion/core": "^10.0.27",
+    "@emotion/styled": "^10.0.27",
+    "adal-node": "0.2.1",
+    "archiver": "^5.0.2",
+    "axios": "^0.21.1",
+    "encoding": "^0.1.13",
+    "fs-extra": "8.1.0",
+    "https-proxy-agent": "^5.0.0",
+    "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.20",
+    "md5": "2.2.1",
+    "minimist": "1.2.5",
+    "office-ui-fabric-react": "^7.145.0",
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0",
+    "recoil": "^0.3.1",
+    "request": "2.88.2",
+    "request-promise": "4.2.5",
```

Comment:
We should dump request-promise in favor of axios.  This library is deprecated and has security flaws.